Model/Paypal.php

```diff
@@ -421,7 +421,7 @@ protected function _buildCheckoutRequest($payment, $amount)
         $request['MerchantId'] = $this->reachHelper->getMerchantId();
         $request['ReferenceId'] = $order->getIncrementId();
         $request['ConsumerCurrency']= $order->getOrderCurrencyCode();
-        $order->getOrderCurrencyCode();
+        $order->getOrderCurrencyCode(); //What it is for?
```

Comment:
Could you explain the difference between PayPalManagement and Paypal? They look like they're doing the same thing and I'm not sure why we need two different files for Paypal

Reply:
In January when I started here I was asked the same question and I still have it. Honestly I believe the the code should be  refactored. During dealing with Mag-93 the discussion outcome was PayPal code is half done by previous vendor, and should be dealt with as part of separate ticket.

Reply:
Ok no problem, thank you!